scripts/oligomap_output_to_sam_nh_filtered.py

```diff
@@ -0,0 +1,450 @@
+#!/usr/bin/env python
+
+"""Transform oligomap output FASTA file to SAM keeping the best alignments.
+
+Read the input FASTA file, pick the best alignment(s) for each read (i.e. all
```

Comment:
The `oligomap` output is a proprietory/custom file format. It is NOT FASTA.

Reply:
Oh! As in the workflow and the `oligomap` documentation the output is always redirected to a  FASTA, I thought that was the format to be set in the output file. Changing that

Reply:
Weird. I'm pretty sure though that it's completely different than FASTA (which is an identifier line starting with `>`, followed by one or more lines representing a DNA, RNA or peptide sequence. But for oligomap we get a complex multiline format representing an alignment.

Please resolve the comment after you have changed it.

Reply:
Yeah, the extension of the output also seemed weird for me but as it seemed to be the way it was set I said nothing.

I will change the output extension on the `oligomap` repository's test data. 
As for the workflow, I'll open an issue to fix that but keep it like that in this PR while changing the docstring data on this script.